content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## User facing changes
+
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis/).
+- ***Added*** the new [Workload Entry](/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** configuration for gateway topology. This addresses sanitizing and providing useful [X-Forward-For headers](https://github.com/istio/istio/issues/7679).
+- ***Added*** experimental [mesh-wide tracing configuration API](/docs/tasks/observability/distributed-tracing/configurability/). This API provides control of trace sampling rates, the [maximum tag lengths](https://github.com/istio/istio/issues/14563) for URL tags, and [custom tags extraction](https://github.com/istio/istio/issues/13018) for all traces within the mesh.
+- ***Added*** experimental [request classification](/docs/tasks/observability/metrics/classify-metrics/) filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.
+- TODO: add a statement around networking v1alpha3 API. Will it be removed in 1.6 or announce deprecation?
+
+## Improvements
```

Comment:
IMO the new structure is fine.
The exception is this heading. "Improvements" is too vague. Readers might ask: Improvement compared to what? The past release? Other projects? What type of improvements? Why is adding an annotation an improvement?
After reading through the notes in the section, it seems to me that a better heading would be "UX changes"
@jacob-delgado WDYT?

Reply:
@rcaballeromx @jacob-delgado I don't think the old and new structures need to be mutually exclusive. The functional area breakdown is an important filter for users only using a subset of Istio features. See my comment on slack.

Reply:
I agree with @frankbu. I just misunderstood the comment Louis made. I'll revert this to the previous structure, but organize by the priority he mentioned.